website/docs/index.html.markdown

```diff
@@ -20,15 +20,16 @@ Use the navigation to the left to read about the available resources.
 
 You can sign up for a Venafi Cloud account by visiting https://www.venafi.com/platform/cloud/devops.
 Once registered, find your API key by clicking your name in the top right of the web interface.  You
-will also need to specify the ID of a `zone` to use when requesting certificates.  Zones are the part 
+will also need to specify the `zone` to use when requesting certificates.  Zones are the part 
 of a Venafi Cloud project that define the machine identity policy applied to certificate requests and
-the certificate authority that will issue certificates.
+the certificate authority that will issue certificates. It's conformed by the Application Name and Issuing 
+Template API Alias (e.g. "Business App\Enterprise CIT").
```

Comment:
You will also need to specify the `zone` to use when requesting certificates.  Zones define the machine identity policy that will be applied to certificate requests and the certificate authority that will issue certificates.  The zone is formed by combining the Application Name and Issuing Template API Alias (e.g. "Business App\Enterprise CIT").

Reply:
Fixed :-)